### ISSUE TITLE
Abdalrazq Altaih : Week 2,   Incremental Development

### ISSUE THREAD

**Issue by Abdalrazaq94** [HackYourFutureBelgium/class-8#101]
### Inspecting the DOC 
- [x]   [And practice studying some HTML/CSS here](https://janke-learning.org/dom-tutor)
- [x]  [Read through this site](https://hackyourfuture.be/inspecting-the-dom)

**HTML & CSS**
- [x]  [What is Responsive Design?](https://www.youtube.com/watch?v=5KGah8bW8GE)

**Code Review**
- [x]  [Take a look through this page from Study](https://study.hackyourfuture.be/programming/code-review)

**Git, GitHub & GitKraken**
- [x]  [git & github for poets](https://www.youtube.com/watch?v=BCQHnlnPusY&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV)
- [x]  [LearnGitBranching ](https://learngitbranching.js.org/)
- [x]  [Familiarize yourself with GitKraken](https://support.gitkraken.com/start-here/interface/)
- [x]  [Study this example project](https://github.com/hackyourfuturebelgium/built-with-branches)
- [x]  [Understand how to use Atomic Commits](https://curiousprogrammer.io/blog/how-to-craft-your-changes-into-small-atomic-commits-using-git)

### Name of Project:
- [x] [repo](https://github.com/Abdalrazaq94/apptheme)
 - [x] [live demo](http://127.0.0.1:5500/index.html)
**branshes**
- [x]  [div center](13f5580b375521dc15a3ea09bfd104ed26e43f07)
- [x]  [font](52f4164c5969012f6b99fde60603894c5c035a91)
- [x]  [css](722e045bedfe0d9155c8ba28f9f2d4a9e57872af)
- [x]  [marge](c577d08f031dc3f50ad6d63ecc1d62e302d595bc)

### Exercises:

Complete all 4 of FreeCodeCamp's Responsive Web Design Principles challenges.

- [x] exercise name: [Complete all 4 of FreeCodeCamp](https://www.freecodecamp.org/learn)
- [x] exercise name: [sceen shot of free camp](
https://scontent-lga3-1.xx.fbcdn.net/v/t1.15752-9/83374607_1081105265595940_5568817023619694592_n.jpg?_nc_cat=102&_nc_ohc=zvpL1XKiLnAAX-wfqFA&_nc_ht=scontent-lga3-1.xx&oh=a66d18e5d9ae26c7e9c397f82905d9e7&oe=5EDB153B
)
**css and html pracrice problem**
- [x]  [repo](https://github.com/Abdalrazaq94/HTML-CSS-Practice-Problems/commit/a617d93b0b985f00c99818deeab1f8439cfe14e4)
- [x]  [live](http://127.0.0.1:5500/index.html)
### CSS **exercise**
- [ ]  [3](http://127.0.0.1:5500/index.html)
- [x]  [repo 4](https://github.com/Abdalrazaq94/css-exercises/tree/master/exercise-4)
 [live 4](http://127.0.0.1:5500/css-exercises/exercise-4/index.html)
- [x]  [5 repo](https://github.com/Abdalrazaq94/css-exercises/tree/master/exercise-5)
  [5 live](http://127.0.0.1:5500/css-exercises/exercise-5/index.html)
i did not know how to make the right branches of my project 
> [Follow this link](https://github.com/HackYourFutureBelgium/class-6/issues/59) to study a well-done homework issue.

**Comment by Abdalrazaq94** [comment]
it is ready for doing the review , i did not know how to make the right branches

**Comment by otagi** [comment]
Good code overall. 👍 

Don't forget to indent your elements properly, both in HTML and CSS. You can do that either manually, or by using the VSCode plugin. 

And yes, you created one branch, and should have merged it in master after 1 or 2 commits, instead of continuing on that branch. 

Also, the link to your live demo in this issue points to `127.0.0.0:5500`, which is you local server. It should point to you project on `github.io` (the GitHub Pages server).
